hasSymbols.js

```diff
@@ -3,5 +3,17 @@ var keys = require('object-keys');
 module.exports = function hasSymbols() {
 	if (typeof Symbol !== 'function' || typeof Object.getOwnPropertySymbols !== 'function') { return false; }
 	if (typeof Symbol.iterator === 'symbol') { return true; }
-	return false;
+	var obj = {};
+	var sym = Symbol('test');
+	if (typeof sym === 'string') { return false; }
```

Comment:
Based on the outcome of the discussion below, either I'll remove this line, or the gOPS polyfill will adapt to return a `String` object.